src/run/runner/helpers/introspected_golang/go.sh

```diff
@@ -11,6 +11,13 @@ debug_log() {
 debug_log "Called with arguments: $*"
 debug_log "Number of arguments: $#"
 
+
+# Currently only walltime is supported
+if [ "${CODSPEED_RUNNER_MODE:-}" != "walltime" ]; then
+    echo "WARNING: Go benchmarks can only be run with walltime"
```

Comment:
It's an error since we're exiting and we can tweak a bit the wording
```suggestion
    echo "CRITICAL: Go benchmarks can only be run with the walltime instrument"
```